fhir-model/src/test/java/com/ibm/fhir/model/spec/test/R4ExamplesDriver.java

```diff
@@ -261,17 +262,15 @@ public void processExample(String file, Format format, Expectation expectation)
                 // If we parsed the resource successfully but expected it to fail, it's a failed
                 // test, so we don't try and process it any further
                 actual = Expectation.OK;
-                ExampleProcessorException error =
-                        new ExampleProcessorException(file, expectation, actual);
-                if (firstException == null) {
-                    firstException = error;
-                }
-                throw error;
+                throw new ExampleProcessorException(file, expectation, actual);
             } else {
                 // validate and process the example
                 actual = processExample(file, resource, expectation);
             }
         } catch (ExampleProcessorException e) {
+            if (firstException == null) {
```

Comment:
What is the purpose of changing the logic here?

Reply:
I wasn't sure why we weren't getting firstException set.
I realized that if processExample (line 268) threw an ExampleProcessorException then we'd never set it.
So I took the logic from the old lines 266-268 and moved them down here instead so that we apply them no matter where ExampleProcessorException is thrown from.

Reply:
It turned out not to be our issue, but I thought the change was good and so I left it.